src/lib.rs

```diff
@@ -476,6 +476,52 @@ impl<T: Debug + PrimInt + One + Zero> Vob<T> {
         }
     }
 
+    /// Returns contents of Vob as Vec of bytes.
+    ///
+    /// For a Vob of length n, the first bit will be the 0th element,
+    /// the last bit would be the (n-1)th element.
+    ///
+    /// # Examples
+    /// ```
+    /// use vob::Vob;
+    /// let mut x = Vob::from_elem(5, false);
+    /// x.set(2, true);
+    /// x.set(4, true);
+    /// assert_eq!(x.to_bytes(), [0b00101000]);
+    /// ```
+    pub fn to_bytes(&self) -> Vec<u8> {
+        let mut v: Vec<u8> = Vec::new();
+        for j in 0..self.vec.len() {
+            let current_block = self.vec[j];
+            let bytes_per_block = bytes_per_block::<T>();
+            for i in (0..bytes_per_block).rev() {
+                let x = current_block
+                    & num_traits::cast::cast(
+                        255_u64.rotate_left(8 * (bytes_per_block - 1 - i) as u32),
+                    )
+                    .unwrap();
+                let y: u8 = num_traits::cast::cast(x >> 8 * (bytes_per_block - 1 - i)).unwrap();
+                if y != 0 {
```

Comment:
This condition should probably be moved inside the `#[cfg(not(reverse_bits))]` clause. It *probably* slows things down for the `#[cfg(reverse_bits)]` case but, again, we should benchmark this.